sql/moz-fx-data-shared-prod/contextual_services_derived/event_aggregates_suggest_v1/query.sql

```diff
@@ -1,47 +1,32 @@
 WITH combined AS (
   SELECT
-    context_id,
+    metrics.uuid.quick_suggest_context_id AS context_id,
     DATE(submission_timestamp) AS submission_date,
     'desktop' AS form_factor,
     normalized_country_code AS country,
-    LOWER(advertiser) AS advertiser,
+    LOWER(metrics.string.quick_suggest_advertiser) AS advertiser,
     SPLIT(metadata.user_agent.os, ' ')[SAFE_OFFSET(0)] AS normalized_os,
-    release_channel,
-    position,
-    IF(request_id IS NULL, 'remote settings', 'merino') AS provider,
-    match_type,
+    client_info.app_channel AS release_channel,
+    metrics.quantity.quick_suggest_position AS position,
+    IF(
+      NULLIF(metrics.string.quick_suggest_request_id, "") IS NULL,
+      'remote settings',
+      'merino'
+    ) AS provider,
```

Comment:
@jmsilverman do you know if we want to treat empty string the same as null here

Reply:
Good question! I found that this was a change from PingCentre (request_id was never NULL but in Glean it's very often "" but also occasionally NULL). I have pointed this out as a potential issue but haven't gotten an official confirmation that we should treat "" and NULL the same. @quiiver and @rebecca-burwei might have more to say about this.

Reply:
FWIW, it looks like in further conversations we've decided that (for now at least) we should be treating "" and NULL as the same.
Related bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1862660